cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -166,91 +166,94 @@ <h2 class="h4">
-                                            <p>Services</p>
-                                        </div>
+                                    <th>
+                                        <p>Services</p>
                                     </th>
-                                    <th scope="col">
-                                        <div class="rich-text">
-                                            <p>Distance</p>
-                                        </div>
+                                    <th>
+                                        <p>Distance</p>
                                     </th>
                                 </tr>
                             </thead>
                             <tbody>
                                 {% for counselor in api_json.counseling_agencies %}
                                 <tr id="hud-result-{{ loop.index }}">
                                     <td data-label="Agency">
-                                        <div class="rich-text">
-                                            <p>
-                                                <span class="result-number">{{ loop.index }}.</span>
-                                                <a class="a-link a-link__icon" href="{{ counselor.weburl }}">
-                                                  <span class="a-link_text">{{ counselor.nme }}</span>
-                                                  {{ svg_icon( 'external-link' ) }}
-                                                </a>
-                                                <br>
-                                                {{ counselor.adr1 }}
-                                                {% if counselor.adr2 and counselor.adr2 != ' '  %}
-                                                <br>
-                                                {{ counselor.adr2 }}
-                                                {% endif %}
-                                                <br>
-                                                {{ counselor.city }}, {{ counselor.statecd }} {{ counselor.zipcd }}
-                                            </p>
-                                            <p>
-                                                {% if counselor.weburl %}
-                                                <b>Website:</b>
-                                                <a class="a-link a-link__icon" href="{{ counselor.weburl }}">
-                                                    <span class="a-link_text">{{ counselor.weburl }}</span>
-                                                    {{ svg_icon( 'external-link' ) }}
-                                                </a>
-                                                <br>
-                                                {% endif %}
+                                        <ol start="{{ loop.index }}">
+                                            <li>
+                                                <p>
```

Comment:
This `<p>` seems unnecessary, and this might be out of scope for this PR, but we should consider marking up everything within these `<td>`s with an address microformat like [h-card](http://microformats.org/wiki/h-card) or the [Organization schema](https://schema.org/Organization). (I am not clear on which of these is the best practice currently...)

Reply:
I believe they are for the gap between the title/address and the website/phone/etc. I'll address further in your other comment.

Reply:
Yeah, I figured that was the purpose, but it's not good semantics. Converting the whole shootin' match to a microformat/schema would give you multiple hooks you could use to add the styling with Less (and it also might come for free with the `<dl>`).

Reply:
[GHE]/CFGOV/platform/issues/3200